src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1897,9 +1898,13 @@ public static void main(String[] args) {
       vleft.connect("COM3");
       vright.connect("COM4");
       Runtime.start("gui", "SwingGui");
```

Comment:
Why are we starting the swing gui here?  (also python?)

Reply:
main() section for testing